_config.yml

```diff
@@ -1030,3 +1030,10 @@ podcast:
           url: "https://github.com/orta"
         - title: "Orta on Twitter"
           url: "https://twitter.com/orta"
+    - title: "9: WAYAHDYGH: Sam"
+      date: "2021-03-17 10:37:02 -0600"
+      description: ""
```

Comment:
@annacarey we should add a description here! this is what will show up in apple podcasts/spotify/etc with the episode

Reply:
OH thank you good catch!